diff --git a/node_modules/@electron-forge/publisher-github/dist/PublisherGithub.js b/node_modules/@electron-forge/publisher-github/dist/PublisherGithub.js
index 785323a..add358c 100644
--- a/node_modules/@electron-forge/publisher-github/dist/PublisherGithub.js
+++ b/node_modules/@electron-forge/publisher-github/dist/PublisherGithub.js
@@ -73,11 +73,16 @@ class PublisherGithub extends publisher_base_1.PublisherBase {
                 flatArtifacts.push(...artifact.artifacts);
             }
             await Promise.all(flatArtifacts.map(async (artifactPath) => {
+                          // standardize the path to artifacts across platforms
+                const normalizedPath = artifactPath.split(/\/|\\/).join(path.sep);
+                if (!(await fs_extra_1.default.pathExists(normalizedPath))) {
+                    throw new Error(`Attempted to resume a dry run but an artifact (${normalizedPath}) could not be found`);
+                }
                 const done = () => {
                     uploaded += 1;
                     updateUploadStatus();
                 };
-                const artifactName = path_1.default.basename(artifactPath);
+                const artifactName = path_1.default.basename(normalizedPath);
                 // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
                 if (release.assets.find((asset) => asset.name === artifactName)) {
                     return done();
@@ -90,12 +95,12 @@ class PublisherGithub extends publisher_base_1.PublisherBase {
                     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
                     url: release.upload_url,
                     // https://github.com/octokit/rest.js/issues/1645
-                    data: (await fs_extra_1.default.readFile(artifactPath)),
+                    data: (await fs_extra_1.default.readFile(normalizedPath)),
                     headers: {
-                        'content-type': mime_types_1.default.lookup(artifactPath) || 'application/octet-stream',
-                        'content-length': (await fs_extra_1.default.stat(artifactPath)).size,
+                        'content-type': mime_types_1.default.lookup(normalizedPath) || 'application/octet-stream',
+                        'content-length': (await fs_extra_1.default.stat(normalizedPath)).size,
                     },
-                    name: path_1.default.basename(artifactPath),
+                    name: path_1.default.basename(normalizedPath),
                 });
                 return done();
             }));
